data/json/items/melee/swords_and_blades.json

```diff
@@ -180,7 +180,7 @@
     "type": "TOOL",
     "category": "weapons",
     "name": { "str": "RM42 fighting knife", "str_pl": "RM42 fighting knives" },
-    "description": "This sturdy matte black Rivtech combat dagger features a long and slim double-edged blade with a spear-point and a distinctive slip-resistant grip which can also be used to affix it to a suitable firearm.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
+    "description": "This sturdy matte black Rivtech combat dagger features a long, slim double-edged blade with a spear point and a distinctive slip-resistant grip, which can also be used to affix it to a suitable firearm.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
```

Comment:
```suggestion
    "description": "This sturdy, matte-black Rivtech combat dagger features a long, slim double-edged blade with a spear point and a distinctive slip-resistant grip, which can also be used to affix it to a suitable firearm as a bayonet.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
```

Reply:
I thought about this, but then I considered that I wouldn't hyphenate "glossy white" in similar circumstances.

Reply:
I'll manually commit "as a bayonet," though.

Reply:
I think I would hyphenate "glossy white", actually, if it were together with anything else. But I probably tend to over-hyphenate; no problem. Comma after "sturdy"?